### PR TITLE
[PB-3397]: feat/switch-to-vitest

### PR DESCRIPTION
**Pull request by TamaraFinogina**
Switch all tests from jest to vitest

### CODE REVIEW

File: src/app/auth/services/auth.service.test.ts [hunk 1]
```diff
@@ -3,19 +3,15 @@
...
-import localStorageService from 'app/core/services/local-storage.service';
-import * as keysService from 'app/crypto/services/keys.service';
+import localStorageService from '../../core/services/local-storage.service';
+import * as keysService from '../../crypto/services/keys.service';
 import { AuthenticateUserParams } from './auth.service';
-import { AuthMethodTypes } from 'app/payment/types';
+import { AuthMethodTypes } from '../../payment/types';
+import { vi, describe, it, expect, beforeEach, afterEach } from 'vitest';
 
-jest.mock('../../../WebWorker');
```
Comment (TamaraFinogina) [inline src/app/auth/services/auth.service.test.ts:13]: @sg-gs , I don't know how to convert this test to vitest yet. It fails the moment any call to authService appears. Seems to be something with the mock, but I'm not sure what. @xabg2  tried as well, but so far it's not working

**Review by sg-gs — APPROVED** [review]

**Comment by sg-gs** [comment]
As this is a switch of the main testing tool: 
- The build is working
- The CI is working
- The analysis (except the testing coverage) is passing

Should we run some additional QA or is this ready for deployment? @CandelR

**Comment by CandelR** [comment]
> As this is a switch of the main testing tool:
> 
> * The build is working
> * The CI is working
> * The analysis (except the testing coverage) is passing
> 
> Should we run some additional QA or is this ready for deployment? @CandelR

@sg-gs In principle it is not necessary, since apart from the unit tests, the files have basically been changed paths. 
But it is not superfluous with so many changes in routes to do a quick QA going through the app to see that there is no unexpected crash due to a bad import, since the e2e are not yet deployed and it is not checked that at least the basic functions and screens work correctly.
I would deploy it in Staging and have a QA done today to deploy this tomorrow.

**Comment by sg-gs** [comment]
Okay @CandelR, meanwhile, @TamaraFinogina, you can create a branch from this branch so when @CandelR merges, you just have to merge master against your branch and remove the 6/7 thousand additions removals you currently have on this PR.

**Comment by CandelR** [comment]
Changed to vitest here: https://github.com/internxt/drive-web/pull/1365

**Pull request closed without merging**